### PR TITLE
[TRANSLATION] Update Chinese translation.

### PR DESCRIPTION
**Pull request by njlyf2011**

### CODE REVIEW

**Comment by njlyf2011** [comment]
Corrected.
Is there a way to fix this grammatical problems with one click? The original translator had a lot of grammatical problems when translating the OS, and it would be cumbersome for me to correct it.

File: dll/win32/tapiui/lang/zh-TW.rc [hunk 1]
```diff
@@ -58,8 +58,8 @@ BEGIN
     LTEXT "區號規則確定電話號碼的撥號方式從你當前的地區代碼於其他地區代碼和區號中。", 2000, 7, 7, 238, 16
     LTEXT "區號規則:(&C)", -1, 7, 31, 238, 8
```
[the remark this answers is not in the document]
Comment (katahiromz) [reply dll/win32/tapiui/lang/zh-TW.rc]:
```suggestion
    LTEXT "區號規則(&C):", -1, 7, 31, 238, 8
```

File: dll/win32/tapiui/lang/zh-TW.rc [hunk 2]
```diff
@@ -70,11 +70,11 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_CAPTION
...
-    LTEXT "卡類型:(&T)", -1, 7, 23, 238, 8
+    LTEXT "選擇您將使用的電話卡，或按新增添加不同的卡。", 2000, 7, 7, 238, 8
+    LTEXT "卡類型(&T):", -1, 7, 23, 238, 8
     CONTROL "", 1033, "SysListView32", WS_BORDER | WS_TABSTOP | 0x0000400D, 7, 34, 238, 63, WS_EX_CLIENTEDGE
-    PUSHBUTTON "新建...(&N)", 1030, 87, 103, 50, 14
-    PUSHBUTTON "編輯...(&E)", 1031, 141, 103, 50, 14
+    PUSHBUTTON "新增(&N)...", 1030, 87, 103, 50, 14
+    PUSHBUTTON "編輯(&E)...", 1031, 141, 103, 50, 14
     PUSHBUTTON "刪除(&D)", 1032, 195, 103, 50, 14
     LTEXT "帳號:(&C)", -1, 7, 127, 91, 8
```
[the remark this answers is not in the document]
Comment (katahiromz) [reply dll/win32/tapiui/lang/zh-TW.rc]:
```suggestion
    LTEXT "帳號(&C):", -1, 7, 127, 91, 8
```

File: dll/win32/tapiui/lang/zh-TW.rc [hunk 3]
```diff
@@ -137,25 +137,25 @@ BEGIN
...
 108 DIALOGEX 0, 0, 252, 218
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_CAPTION
 CAPTION "本地電話"
 FONT 9, "新細明體"
 BEGIN
-    LTEXT "本地電話訪問號碼:(&L)", -1, 7, 7, 238, 8
+    LTEXT "本地電話訪問號碼(&L):", -1, 7, 7, 238, 8
     EDITTEXT 1008, 7, 18, 140, 14, ES_AUTOHSCROLL
     LTEXT "使用下面的按鈕來輸入打市內電話撥號步驟。當他們出現在你的名片上，按準確順序輸入這些步驟。來而無需使用您的電話卡打本地電話，請將此部分留爲空白。", 2000, 7, 40, 238, 24
     LTEXT "電話卡撥號步驟:(&R)", -1, 7, 72, 238, 8
```
[the remark this answers is not in the document]
Comment (katahiromz) [reply dll/win32/tapiui/lang/zh-TW.rc]:
```suggestion
    LTEXT "電話卡撥號步驟(&R):", -1, 7, 72, 238, 8
```

File: dll/win32/tapiui/lang/zh-TW.rc [hunk 4]
```diff
@@ -179,11 +179,11 @@ FONT 9, "新細明體"
...
     LTEXT "此區號規則只適用於電話的區號和您在下面指定的前綴組合作出。", 2000, 7, 6, 238, 16
     LTEXT "要調用的地區代碼:", -1, 7, 31, 110, 8
-    LTEXT "地區代碼:(&R)", -1, 7, 47, 36, 8
+    LTEXT "地區代碼(&R):", -1, 7, 47, 36, 8
     EDITTEXT 1034, 46, 44, 50, 14, ES_AUTOHSCROLL
     LTEXT "指定要用於此區號規則的前綴。", -1, 14, 78, 224, 8
     AUTORADIOBUTTON "包含此區號之內的所有前綴(&I)", 1014, 14, 90, 224, 12, NOT WS_TABSTOP
-    AUTORADIOBUTTON "在下面的列表中包括唯一的前綴:(&O)", 1015, 14, 104, 224, 12, NOT WS_TABSTOP
+    AUTORADIOBUTTON "在下面的列表中包括唯一的前綴(&O):", 1015, 14, 104, 224, 12, NOT WS_TABSTOP
     LTEXT "要包含的前綴:(&P)", 1057, 26, 117, 119, 8
```
[the remark this answers is not in the document]
Comment (katahiromz) [reply dll/win32/tapiui/lang/zh-TW.rc]:
```suggestion
    LTEXT "要包含的前綴(&P):", 1057, 26, 117, 119, 8
```

File: dll/win32/tapiui/lang/zh-TW.rc [hunk 5]
```diff
@@ -179,11 +179,11 @@ FONT 9, "新細明體"
...
     CONTROL "List1", 1033, "SysListView32", WS_BORDER | WS_TABSTOP | 0x0000411D, 25, 128, 66, 32, WS_EX_CLIENTEDGE
     PUSHBUTTON "添加...(&A)", 1005, 95, 128, 50, 14
```
[the remark this answers is not in the document]
Comment (katahiromz) [reply dll/win32/tapiui/lang/zh-TW.rc]:
```suggestion
    PUSHBUTTON "添加(&A)...", 1005, 95, 128, 50, 14
```

**Comment by njlyf2011** [comment]
Fixed.

File: dll/win32/tapiui/lang/zh-TW.rc (hunk 2 above)
[the remark this answers is not in the document]
Comment (katahiromz) [reply dll/win32/tapiui/lang/zh-TW.rc]: You're forgetting at here

[the remark this answers is not in the document]
Comment (katahiromz) [reply dll/win32/tapiui/lang/zh-TW.rc]: Here.

**Comment by njlyf2011** [comment]
Fixed.

**Pull request merged**